### PR TITLE
Release 0.14.1

### PR DESCRIPTION
**Pull request by tombye**
This brings in a set of fixes for various issues hence the patch-level increment.

### CODE REVIEW

**Comment by quis** [comment]
https://github.com/alphagov/govuk_template/compare/v0.14.0%E2%80%A6b05e385

**Comment by dsingleton** [comment]
It's a bit late now, but +1 to reverting https://github.com/alphagov/govuk_template/pull/163.

If we do remove it then the changelog should be a bit more explicit, so anyone using it knows they should remove the override value they're setting. (i think it's unlikely anyone is using it, but we should act in good faith here).

**Comment by quis** [comment]
@dsingleton It's being used here, at least as an example: https://github.com/alphagov/govuk_prototype_kit/pull/15

**Comment by dsingleton** [comment]
> @dsingleton It's being used here, at least as an example: alphagov/govuk_prototype_kit#15

Then +2 to keeping it :) It'd be useful for things like the component-guide and govspeak-preview too.

**Comment by quis** [comment]
@dsingleton https://github.com/alphagov/govuk_template/pull/166

**Comment by dsingleton** [comment]
Merged #166, so the diff against master now looks like: https://github.com/alphagov/govuk_template/compare/v0.14.0%E2%80%A6master

**Comment by quis** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 0.14.1
+
+- update Crown Copyright link to new URL
+- fix Jenkins CI build by adding a path to install gems to
```
Comment (dsingleton) [inline CHANGELOG.md]: I think we can drop this, it's not helpful to users of the gem

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+- revert override for GOV.UK text in logo
```
Comment (dsingleton) [inline CHANGELOG.md]: We can drop this now the revert has been reverted

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# 0.14.1
+
+- update Crown Copyright link to new URL
```
Comment (dsingleton) [inline CHANGELOG.md]: Looking at the diff between the last version and master (https://github.com/alphagov/govuk_template/compare/v0.14.0%E2%80%A6master) i don't see the link change?

**Comment by dsingleton** [comment]
:+1: thank you for the persistence!

**Comment by tombye** [comment]
Thanks for the help :)

**Comment by quis** [comment]
:fireworks:

**Pull request merged**